tests/integration/schema/test_configs.py

```diff
@@ -20,24 +22,54 @@
 models_config_path = config_path / "models"
 datasets_config_path = config_path / "datasets"
 experiments_config_path = config_path / "experiments"
+augmentation_config_path = config_path / "augmentations"
 
 models_config_files = [[item] for item in models_config_path.rglob("*.yaml")]
 datasets_config_files = []
-experiment_config_files = [[item] for item in experiments_config_path.rglob("*.yaml")]
+experiment_config_files = [
+    [item] for item in experiments_config_path.rglob("*.yaml")
+]
+augmentation_config_files = []
+augmentation_keys = (
```

Comment:
not sure if this variable is needed. You could store pathlib.Path objects in augmentation_config_files and for each item get its parent name: 
`[i.parent.name for i in augmentation_config_files]`